cmake/modules/BuildSPDK.cmake

```diff
@@ -34,13 +34,7 @@ macro(build_spdk)
   endif()
 
   set(source_dir "${CMAKE_SOURCE_DIR}/src/spdk")
-  set (options lvol env_dpdk sock nvmf bdev nvme conf thread trace notify accel event_accel blob vmd event_vmd event_bdev sock_posix event_sock event rpc jsonrpc json util log)
-  set(configure_command ./configure --with-dpdk=${DPDK_DIR} --without-isal --without-vhost --target-arch=${target_arch})
-  if(WITH_RDMA)
-    list(APPEND options "rdma")
-    list(APPEND configure_command "--with-rdma=mlx5_dv")
-  endif()
-  foreach(c ${options})
+  foreach(c lvol env_dpdk sock nvmf bdev nvme conf thread trace notify accel event_accel blob vmd event_vmd event_bdev sock_posix event_sock event rpc jsonrpc json util log)
```

Comment:
i don't really understand this change. could please explain how it is connected to 

> we have corrected the cmake options

?

Reply:
Because you suggested "add a cmake option to build spdk with rdma" in the last comment of #57833, so we can choose whether to enable rdma when adding the spdk compilation option.